### PR TITLE
Adding echo to run-e2e script

### PR DESCRIPTION
**Pull request by krzysied**
For debugging purpose.

### CODE REVIEW

File: run-e2e.sh [hunk 1]
```diff
@@ -29,6 +29,7 @@ case "$1" in
     ;;
   cluster-loader2 )
     #CLUSTERLOADER2
+    echo "${PERFTEST_ROOT}/clusterloader2 && ./run-e2e.sh ${@:2}"
```
Comment (shyamjvs) [inline run-e2e.sh]: Maybe you should add an echo above the switch case (in case we aren't even entering here).

Comment (krzysied) [reply run-e2e.sh]: Done.

**Pull request merged**